Formula/q/qman.rb

```diff
@@ -0,0 +1,64 @@
+class Qman < Formula
+  desc "Modern man page viewer"
+  homepage "https://github.com/plp13/qman"
+  url "https://github.com/plp13/qman/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "785441bf11e99ff27869c08f0d36ce3f5c75db1b045b8712fe515059cf396780"
+  license "BSD-2-Clause"
+
+  depends_on "cogapp" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "groff"
+  depends_on "ncurses"
+
+  on_linux do
+    depends_on "libbsd"
+  end
+
+  def install
+    inreplace %w[meson_options.txt src/config.h.cog src/config.c.cog man/qman.1],
+              "/etc/xdg/", etc/"xdg/"
+    args = %w[
+      -Dtests=disabled
+      -Dlzma=disabled
+    ]
+    args += if OS.linux?
+      %W[
+        -Dlibbsd=enabled
+        -Dpkg_config_path=#{Formula["libbsd"].opt_lib}/pkgconfig:#{Formula["ncurses"].opt_lib}/pkgconfig
+      ]
+    else
+      %W[
+        -Dpkg_config_path=#{Formula["ncurses"].opt_lib}/pkgconfig
+      ]
+    end
```

Comment:
```suggestion
    args = %W[
      -Dtests=disabled
      -Dlzma=disabled
      -Dpkg_config_path=#{ENV["PKG_CONFIG_PATH"]}
    ]
    args += %w[-Dlibbsd=enabled] if OS.linux?
```
Not sure why we need `-Dpkg_config_path`, though, given that we set `PKG_CONFIG_PATH` in the environment already.